src/components/Button.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from './Button.module.css';
+
+const Button = props => {
+  const { name, width = '25%', color = 'orange' } = props;
```

Comment:
[optional] instead of defining the default value of width and color like this, use the defaultprops functionality in react

Reply:
done